source/Command.cpp

```diff
@@ -272,45 +272,49 @@ void Command::Load(const DataNode &node)
 // Reset this to an empty command.
 void Command::Clear()
 {
-	*this = Command();
+	state = 0;
+	weapon.clear();
+	aim.clear();
 }
 
 
 
 // Clear any commands that are set in the given command.
-void Command::Clear(Command command)
+void Command::Clear(const Command &command)
 {
 	state &= ~command.state;
 }
 
 
 
 // Set any commands that are set in the given command.
-void Command::Set(Command command)
+void Command::Set(const Command &command)
 {
 	state |= command.state;
```

Comment:
This function originally also set fire bits. Should it still do so? (I feel that it should.)